include/blmc_robots/one_joint_driver.hpp

```diff
@@ -24,45 +20,45 @@ namespace blmc_robots
 class OneJointDriver : public NJointBlmcRobotDriver<1, 1>
 {
 public:
-    /**
-     * @brief Constructor
-     *
-     * @param can_port  Name of the CAN port to which the joint is connected
-     *     (e.g. "can0")
-     * @param home_offset_rad  Home offset of the joint.  This is the offset
-     *     between the home position (= encoder index found during homing) and
-     *     the desired zero position.
-     */
-    OneJointDriver(const std::string &can_port,
-                   const double home_offset_rad = 0.0)
-        : OneJointDriver(create_motor_boards({can_port}), home_offset_rad)
+    static Config get_default_config()
```

Comment:
as mentioned above, i think it might be a good idea to get rid of all the default config functions..